_episodes/01-intro.md

```diff
@@ -26,8 +26,8 @@ At a high level, computers do four things:
 -   communicate with each other, and
 -   interact with us
 
-They can do the last of these in many different ways,
-including through a keyboard and mouse, or touch screen interfaces, or speech recognition using systems.
+They can do the last of these in many different ways.
+These interactions can occur through a keyboard and mouse, a touch screen interfaces, or with systems capable of speech recognition.
```

Comment:
The `a` can be dropped in the following line:

> a touch screen interfaces

Reply:
Yep, I made a typo in my correction!

Reply:
Can you fix this an push an update to fix that? Thanks.